torch/csrc/jit/passes/quantization.cpp

```diff
@@ -667,6 +667,20 @@ class InsertObserversHelper {
   // These are the IR patterns we match to skip inserting observers.
   // They are compiled once on construction and used repeatedly within
   // the pass.
+  const PatternInfo linear_functional_relu = PatternInfo::parse_from_str(R"(
```

Comment:
maybe add the fusion for F.linear + F.relu and F.linear + self.relu(module) here?
We may need these two patterns for F.conv as well, but it can be in a different PR.

Reply:
OK, I will look into this